specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json

```diff
@@ -478,6 +478,17 @@
           "format": "int64",
           "description": "Offset for consumer record",
           "readOnly": true
+        },
+        "timestamp": {
```

Comment:
add properties in response is considered as breaking change, https://github.com/Azure/adx-documentation-pr/wiki/Breaking-changes-guidelines#new-property-added-to-response, suggest to add it in new API version.

Reply:
This feature is still in preview and no users yet. This is a recommendation from  a potential user who might consider to use the api if those properties exist. Hence, it is okay not to add new api version for this.

Reply:
thanks for update